#83 Feature- implement notification panel

Shumba250 opened the pull request:
#### What does this PR do?
The notifications panel displays a user's notifications and prompts them to take action when they interact with notification icon
#### Description of Task to be completed

- The notifications pane should open up when a user clicks on the icon. It should contain a list of notifications
#### How should this be manually tested?
clone the repo using git clone repo's URL
change branch git ft- implement notification panel
create .env file and paste variables from .env.example
run npm run dev
login as a vendor
You should be redirected to the dashboard
click on the notification icon and you should be able to see the notification

Shumba250 committed:
ft-create-vendor-layout
implement dropout sidebar
implement navigation bar
implement links to products, sales and emplyees page
implement fully responsive dashboard

Shumba250 committed:
ft-create-vendor-layout
implement dropout sidebar
implement navigation bar
implement links to products, sales and emplyees page
implement fully responsive dashboard

Shumba250 commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Implement notifications pane](https://trello.com/c/6O2pHIzb/83-implement-notifications-pane)

Shumba250 committed:
feature(implement notification panel):
The notifications pane should open up when a user clicks on the icon. It should contain a list of notifications

Shumba250 committed:
feature(implement notification panel):
The notifications pane should open up when a user clicks on the icon. It should contain a list of notifications

sostene12 reviewed (approved):
<img width="1260" alt="Screenshot 2023-05-15 at 17 50 04" src="https://github.com/atlp-rwanda/ecommerce-app-champions-fe/assets/88575509/0d11274d-d513-4f45-9211-2164dc16a695">
the notifications are working, when i make some action, but the style of products are not verywell, consider improving it

wakaflorien commented on vite.config.js.timestamp-1684135706373-7f1bd4242e6e6.mjs:
```diff
@@ -0,0 +1,53 @@
+	},
+});
+export { vite_config_default as default };
+//# sourceMappingURL=data:application/json;base64,ewogICJ2ZXJzaW9uIjogMywKICAic291cmNlcyI6IFsidml0ZS5jb25maWcuanMiXSwKICAic291cmNlc0NvbnRlbnQiOiBbImNvbnN0IF9fdml0ZV9pbmplY3RlZF9vcmlnaW5hbF9kaXJuYW1lID0gXCJDOlxcXFxVc2Vyc1xcXFxUR1RcXFxcRGVza3RvcFxcXFxzb3N0ZW5lXFxcXGVjb21tZXJjZS1hcHAtY2hhbXBpb25zLWZlXCI7Y29uc3QgX192aXRlX2luamVjdGVkX29yaWdpbmFsX2ZpbGVuYW1lID0gXCJDOlxcXFxVc2Vyc1xcXFxUR1RcXFxcRGVza3Rvc ...
```
- check the vite config file.js the commented section

wakaflorien commented on src/main.jsx:
```diff
@@ -1,14 +1,23 @@
+import { ContextProvider } from './contexts/ContextProvider';
+
+registerLicense(
+	'Mgo+DSMBaFt+QHJqVk1mQ1JbdF9AXnNNdFB3T2Zebz4Nf1dGYl9RQXZbQ1tgSHtUf0xnWQ==;Mgo+DSMBPh8sVXJ1S0R+X1pCaVxdX2NLfUNyT2BYdVt3ZDU7a15RRnVfR11nSHhQd0BrWX5ecA==;ORg4AjUWIQA/Gnt2VFhiQlJPcEBBQmFJfFBmQWlaflR1c0UmHVdTRHRcQlhjTX9VdURmUX9ceHQ=;MTkyNjU1NkAzMjMxMmUzMjJlMzNPYy9kbVlub2xtaHZPeTcrQTQydCtoN29wRWNWd3I4ejNFanIwWmJNMklvPQ==;MTkyNjU1N0AzMjMxMmUzMjJlMzNlS290R2VBK0lsTmt5amdldjhUcXB0TTFjbXRTOUgrTVZhUis1c3JqW ...
```
-  ass license to .env file

Shumba250 committed:
feature(implement notification panel):
The notifications pane should open up when a user clicks on the icon. It should contain a list of notifications
